samples/ControlCatalog/ControlCatalogControl.xaml.cs

```diff
@@ -0,0 +1,19 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Markup.Xaml;
+
+namespace ControlCatalog
+{
+    public class ControlCatalogControl : UserControl
```

Comment:
Not sure about the name of this, `ControlCatalog.ControlCatalogControl ` is a bit unwieldy. Something like `ControlCatalog.MainView` would be better IMHO